pff_equipment_testing/src/pff_equipment_testing/config/hw_setup.yaml

```diff
@@ -0,0 +1,16 @@
+# Global params
+loop_rate: 100
+log_location: /home/jwhite/bldc/pff_equipment_testing/src/pff_equipment_testing/
```

Comment:
this path is going to break on any computer thats not yours

Reply:
I changed this, but if the specified path doesn't exist the log server will realize and default to ~/.ros/